tests/integration/targets/ndo_tenant_igmp_interface_policy/tasks/main.yml

```diff
@@ -0,0 +1,558 @@
+        version3_asm: 'enabled'
+        fast_leave: 'enabled'
+        report_link_local_groups: 'enabled'
+        igmp_version: 'v3'
+        group_timeout: 300
+        query_interval: 150
+        query_response_interval: 20
+        last_member_count: 3
+        last_member_response_time: 12
+        startup_query_count: 4
+        startup_query_interval: 1000
+        querier_timeout: 700
+        robustness_variable: 6
+        state_limit_route_map:
+          name: ansible_test_route_map_policy_1
+        report_policy_route_map:
+          name: ansible_test_route_map_policy_2
+        static_report_route_map:
+          name: ansible_test_route_map_policy_3
+        maximum_multicast_entries: 3244545
+        reserved_multicast_entries: 8635434
+        state: present
+      register: create_igmp_interface_policy_2
+
+    - name: Assert IGMP Interface Policy 2 was created
+      assert:
+        that:
+          - create_igmp_interface_policy_2 is changed
+          - create_igmp_interface_policy_2.previous == {}
+          - create_igmp_interface_policy_2.current.name == 'ansible_igmp_interface_policy_2'
+          - create_igmp_interface_policy_2.current.templateName == 'ansible_tenant_template'
+          - create_igmp_interface_policy_2.current.templateId is defined
+          - create_igmp_interface_policy_2.current.uuid is defined
+          - create_igmp_interface_policy_2.current.description == 'IGMP Interface Policy 2 description'
+          - create_igmp_interface_policy_2.current.enableFastLeaveControl == true
+          - create_igmp_interface_policy_2.current.enableReportLinkLocalGroups == true
+          - create_igmp_interface_policy_2.current.enableV3Asm == true
+          - create_igmp_interface_policy_2.current.groupTimeout == 300
+          - create_igmp_interface_policy_2.current.igmpQuerierVersion == 'v3'
+          - create_igmp_interface_policy_2.current.lastMemberCount == 3
+          - create_igmp_interface_policy_2.current.lastMemberResponseInterval == 12
+          - create_igmp_interface_policy_2.current.maximumMulticastEntries == 3244545
+          - create_igmp_interface_policy_2.current.querierTimeout == 700
+          - create_igmp_interface_policy_2.current.queryInterval == 150
+          - create_igmp_interface_policy_2.current.queryResponseInterval == 20
+          - create_igmp_interface_policy_2.current.reservedMulticastEntries == 8635434
+          - create_igmp_interface_policy_2.current.robustnessFactor == 6
+          - create_igmp_interface_policy_2.current.startQueryCount == 4
+          - create_igmp_interface_policy_2.current.startQueryInterval == 1000
+          - create_igmp_interface_policy_2.current.reportPolicyRouteMapName == 'ansible_test_route_map_policy_2'
+          - create_igmp_interface_policy_2.current.stateLimitRouteMapName == 'ansible_test_route_map_policy_1'
+          - create_igmp_interface_policy_2.current.staticReportRouteMapName == 'ansible_test_route_map_policy_3'
```

Comment:
are we setting the uuid in the output? if not should we, else should we check if defined in the test case?

Reply:
Added  UUID as defined